great_expectations/experimental/datasources/dynamic_pandas.py

```diff
@@ -0,0 +1,284 @@
+import inspect
```

Comment:
@Kilo59 Since this module does not define a `Datasource` (unless I did not notice that class definition), then perhaps we can call it something like `dynamic_pandas_support.py` (or `dynamic_pandas_helper.py`), etc.?  Thanks.